packages/orbit-components/transforms/tokens-v16.mjs

```diff
@@ -128,15 +128,15 @@ const spacingTransformations = [
   { from: /(size|w|h)-xxl/, to: "$1-1000" },
   { from: /(size|w|h)-xxxl/, to: "$1-[52px]" },
   // tailwind theme access
-  { from: /theme\(spacing.xxxs\)/, to: "theme(spacing.50)" },
-  { from: /theme\(spacing.xxs\)/, to: "theme(spacing.100)" },
-  { from: /theme\(spacing.xs\)/, to: "theme(spacing.200)" },
-  { from: /theme\(spacing.sm\)/, to: "theme(spacing.300)" },
-  { from: /theme\(spacing.md\)/, to: "theme(spacing.400)" },
-  { from: /theme\(spacing.lg\)/, to: "theme(spacing.600)" },
-  { from: /theme\(spacing.xl\)/, to: "theme(spacing.800)" },
-  { from: /theme\(spacing.xxl\)/, to: "theme(spacing.1000)" },
-  { from: /theme\(spacing.xxxl\)/, to: "52px" },
+  { from: /theme\((spacing|space)\.xxxs\)/, to: "theme(spacing.50)" },
```

Comment:
@DSil Do you know the difference between `spacing` and `space` here? In our codebase we seem to have references only to `spacing`. Does this match with the [Tailwind theme config](https://github.com/kiwicom/orbit/blob/db70be98696b3a330d1cecc7116f2629351bb704/packages/orbit-tailwind-preset/src/foundation/getTailwindTheme.ts#L129)?

Reply:
Official docs for [spacing](https://tailwindcss.com/docs/customizing-spacing) and [theme function](https://tailwindcss.com/docs/functions-and-directives#theme) mention `spacing` and that is what we use in Orbit. I don't know why others refer to it as `space` even if their config does not mention it 🤷‍♂️ apparently it works. But I don't know if it's some legacy naming that Tailwind kept for retrocompatibility… But I would prefer to just use spacing, tbh

Reply:
> I don't know why others refer to it as space even if their config does not mention it 🤷‍♂️ apparently it works.

Wow, I'm surprised about this. Maybe we should ask them about it.